cmd/copy.go

```diff
@@ -1382,4 +1399,11 @@ func init() {
 	// Hide the flush-threshold flag since it is implemented only for CI.
 	cpCmd.PersistentFlags().Uint32Var(&ste.ADLSFlushThreshold, "flush-threshold", 7500, "Adjust the number of blocks to flush at once on accounts that have a hierarchical namespace.")
 	cpCmd.PersistentFlags().MarkHidden("flush-threshold")
+
+	// White Glove
+	// Get the reporting Webhook UIR
+	// Get the reporting project reference numbers
+	cpCmd.PersistentFlags().StringVar(&raw.reportingAPI, "reporting-api", "", "URI of the Webhook where telemetry events should be posted.")
+	cpCmd.PersistentFlags().StringVar(&raw.projectID, "project-id", "", "The project reference number.")
```

Comment:
ditto to env var or root command parameter, preferably env var considering what you did with cancel